tests/build/test_build_disease.py

```diff
@@ -18,7 +18,7 @@ def test_build_disease_term(adapter, test_disease):
     assert disease_obj["genes"] == [17978]
     assert disease_obj["source"] == "OMIM"
 
-    assert isinstance(disease_obj, DiseaseTerm)
+    assert isinstance(disease_obj, dict)
```

Comment:
I remember that function returning a dictionary because the Pydantic class can't have a _id (or anything with underscore) among its fields, but I'll check it again, thanks!

Reply:
I mean, it can have fields with underscore, but they are ignored..

Reply:
Ah did you intend to switch the imports?
We are at 
```from scout.models.phenotype_term import DiseaseTerm```
both here and on main, so this is presently still a test for the use of the old class `scout.models.phenotype_term.DiseaseTerm`, as it was before.

Reply:
Wait, the DiseaseTerm Class has changed:

Main branch:

<img width="292" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/3c972deb-4389-4be4-bdaf-f26e4cfca098">


This branch:

<img width="304" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/7dbf8fb8-4016-4aaa-8544-0355ca9bb40f">

Reply:
AHHHH Now I see, there are 2 in 2 different files. Yes, confusing. I'll fix!